requirements/requirements.txt

```diff
@@ -39,7 +39,7 @@ pytz==2014.10
 PyYAML==3.10
 raven==3.1.17
 requests==2.0.0
-south==0.7.3
+south==1.0.2
```

Comment:
This will run a migration to add an ApiKey table. the migration in the migrations folder is for django 1.7 and didn't work with south. there's a south migration in south_migrations, but that isn't supported until south 1.0